rules/work/genes/scripts/gnomad_constraints_v4_to_tsv.py

```
@@ -0,0 +1,71 @@
+    )
+    print("No mane_select or canonical transcript found for ", discard.select("gene"))
+    df = (
+        df.filter(pl.col("mane_or_canonical") == True)
+        .rename(columns_src_dst)
+        .with_columns(pl.lit(None).alias(col) for col in additional_columns)
+        .join(ensembl_xlink, on=["ensembl_transcript_id"])
+        .select(columns_dst)
+        .fill_null("NA")
+    )
+    df.write_csv(output_tsv_path, separator="\t")
+    shell(f"md5sum {output_tsv_path} > {output_tsv_md5_path}")
```

Ruff (0.8.2)

Undefined name `shell`

(F821)